packages/react-components/react-components/etc/react-components.api.md

```diff
@@ -1970,6 +1990,10 @@ export { usePortal_unstable }
 
 export { usePresenceBadge_unstable }
 
+export { useProgressBar_unstable }
+
+export { useProgressBarStyles_unstable }
```

Comment:
this shouldnt be _unstable?

Reply:
Hooks are marked with `_unstable` since there might be changes. When releasing a component to stable, we only move the exports from `src/unstable/index.ts` to `src/index.ts`